src/Stratis.Bitcoin.Networks/Policies/StratisStandardScriptsRegistry.cs

```diff
@@ -20,6 +20,7 @@ public class StratisStandardScriptsRegistry : StandardScriptsRegistry
             PayToPubkeyTemplate.Instance,
             PayToScriptHashTemplate.Instance,
             PayToMultiSigTemplate.Instance,
+            PayToFederationTemplate.Instance,
```

Comment:
We will remove this registry at some point anyway, so it doesn't make much difference to add this here

Reply:
Removed.